Apps/Database/src/Migrations/20230428053159_ReworkMessageVerfications.cs

```diff
@@ -0,0 +1,145 @@
+﻿// <auto-generated />
+#pragma warning disable CS1591
+using System;
+using System.Collections.Generic;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace HealthGateway.Database.Migrations
+{
+    /// <inheritdoc />
+    public partial class ReworkMessageVerfications : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropForeignKey(
+                name: "FK_MessagingVerification_Email_EmailId",
+                schema: "gateway",
+                table: "MessagingVerification");
+
+            migrationBuilder.AddColumn<string>(
+                name: "EmailAddress",
+                schema: "gateway",
+                table: "MessagingVerification",
+                type: "character varying(254)",
+                maxLength: 254,
+                nullable: true);
+
+            migrationBuilder.AlterColumn<string>(
+                name: "Subject",
+                schema: "gateway",
+                table: "Email",
+                type: "character varying(100)",
+                maxLength: 100,
+                nullable: true,
+                oldClrType: typeof(string),
+                oldType: "character varying(100)",
+                oldMaxLength: 100);
+
+            migrationBuilder.AlterColumn<string>(
+                name: "Body",
+                schema: "gateway",
+                table: "Email",
+                type: "text",
+                nullable: true,
+                oldClrType: typeof(string),
+                oldType: "text");
+
+            migrationBuilder.AddColumn<Guid>(
+                name: "NotificationId",
+                schema: "gateway",
+                table: "Email",
+                type: "uuid",
+                nullable: true);
+
+            migrationBuilder.AddColumn<Dictionary<string, string>>(
+                name: "Personalization",
+                schema: "gateway",
+                table: "Email",
+                type: "jsonb",
+                nullable: true);
+
+            migrationBuilder.AddColumn<string>(
+                name: "Template",
+                schema: "gateway",
+                table: "Email",
+                type: "character varying(60)",
+                maxLength: 60,
+                nullable: true);
+
+            migrationBuilder.AddForeignKey(
+                name: "FK_MessagingVerification_Email_EmailId",
+                schema: "gateway",
+                table: "MessagingVerification",
+                column: "EmailId",
+                principalSchema: "gateway",
+                principalTable: "Email",
+                principalColumn: "EmailId",
+                onDelete: ReferentialAction.SetNull);
+        }
+
+        /// <inheritdoc />
+        protected override void Down(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropForeignKey(
```

Comment:
Interesting that this would be DropForeignKey and not AddForeignKey, in the auto generated code.